### PR TITLE
{bio}[system/system] nullarbor v2.0.20191013

### PR DESCRIPTION
**Pull request by robqiao**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by robqiao** [comment]
Test report by @robqiao
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
7LN0T92 - Linux UNKNOWN UNKNOWN, Intel(R) Core(TM) i5-4590 CPU @ 3.30GHz, Python 3.8.1
See https://gist.github.com/50eba8ff88e3f7cfda587ffd15b010b3 for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/3fc4cb1747cb11bd6f4f09b4d1402d2a for a full test report.

File: easybuild/easyconfigs/n/nullarbor/nullarbor-2.0.20191013.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+channels = ['conda-forge', 'bioconda']
+
+requirements = '%(name)s=%(version)s python=3.6'
+
+dependencies = [
+    ('Miniconda3', '4.7.10', '', True)
+]
+
+sanity_check_commands = [('nullarbor.pl', '-h')]
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/n/nullarbor/nullarbor-2.0.20191013.eb]: there is `nullarbor.pl --check`, but if fails with missing databases, should those also be installed?

Comment (robqiao) [reply easybuild/easyconfigs/n/nullarbor/nullarbor-2.0.20191013.eb]: @migueldiascosta Thx for pointing `--check` option. I think it's a duplicate feature as `sanity_check` checks dir `perl5` which checks `perl` modules,  `files` checks all binaries. 
You probably are right, a more explicit option can be invoked to check each required `perl` modules. 

A complete `nullarbor` has two parts `framework` and `dataset`, this easyconfig only cater `framework` part. Given the reference `database` is of a reasonable size `24G` altogether and different research may use different reference databases, I would prefer to leave this part to the individual institution. I can append a few lines of instructions at the end, see what you @migueldiascosta think.

**Comment by robqiao** [comment]
Ping @migueldiascosta can we merge this PR? Thx

**Comment by Micket** [comment]
Sorry for the delayed response, with reference to my comment in #10708, we will decline conda-packages in the upstream easyconfig repo.

**Comment by robqiao** [comment]
@Micket 

I might think this one may suit for conda contrib, as the source build is quite finiky on dep chains. See how you think.

**Pull request closed without merging**